docs/tour/add-features golang code broken

Following the Tour from scratch, I get an error compiling in the "adding persistent storage" section.

```
 Building project...
../../../../../../go/pkg/mod/go.wasmcloud.dev/component@v0.0.4/log/wasilog/slog.go:9:2: missing go.sum entry for module providing package github.com/samber/slog-common (imported by go.wasmcloud.dev/component/log/wasilog); to add:
        go get go.wasmcloud.dev/component/log/wasilog@v0.0.4
```

`go.mod` should contain the following entry:

`github.com/samber/slog-common v0.17.1`

--- a/docs/tour/add-features.mdx
+++ b/docs/tour/add-features.mdx
@@ -221,10 +221,11 @@ world hello {
 ```
   </TabItem>
 </Tabs>
-We've given our application the ability to perform atomic incrementation and storage operations via the `wasi:keyvalue` interface and general logging operations via `wasi:logging`.
 
 <Tabs groupId="lang" queryString>
   <TabItem value="tinygo" label="Go">
+
+We've given our application the ability to perform atomic incrementation and storage operations via the `wasi:keyvalue` interface.
 
 Now let's use the atomic increment function to keep track of how many times we've greeted each person.
 
@@ -275,8 +276,16 @@ func handleRequest(w http.ResponseWriter, r *http.Request) {
 func main() {}
 ```
 
+Download the `wasilog` module for idiomatic Go logging that compiles to a component:
+
+```shell
+go get go.wasmcloud.dev/component/log/wasilog
+```
+
   </TabItem>
     <TabItem value="rust" label="Rust">
+
+We've given our application the ability to perform atomic incrementation and storage operations via the `wasi:keyvalue` interface and general logging operations via `wasi:logging`.
 
 Now let's use the atomic increment function to keep track of how many times we've greeted each person.
 
@@ -321,6 +330,8 @@ impl http::Server for Component {
 
   </TabItem>
   <TabItem value="typescript" label="TypeScript">
+
+We've given our application the ability to perform atomic incrementation and storage operations via the `wasi:keyvalue` interface and general logging operations via `wasi:logging`.
 
 Now let's use the atomic increment function to keep track of how many times we've greeted each person.
 

--- a/docs/tour/extend-and-deploy.mdx
+++ b/docs/tour/extend-and-deploy.mdx
@@ -116,7 +116,7 @@ Open `wadm.yaml` in your code editor and make the changes below.
 apiVersion: core.oam.dev/v1beta1
 kind: Application
 metadata:
-  name: hello-world
+  name: hello-world # [!code ++:3]
   annotations:
     description: 'HTTP hello world demo'
 spec:

